changelog_unreleased/markdown/pr-7178.md

```diff
@@ -0,0 +1,79 @@
+#### markdown: fix redundant leading spaces in markdown list ([#7178](https://github.com/prettier/prettier/pull/7178) by [@sasurau4](https://github.com/sasurau4))
```

Comment:
The prefix for the language is unnecessary in changelog. Please fix it in your another PRs too.
```suggestion
#### Fix redundant leading spaces in markdown list ([#7178](https://github.com/prettier/prettier/pull/7178) by [@sasurau4](https://github.com/sasurau4))
```